application/shared-kernel/ApiCore/Middleware/WebAppHandlerMiddleware.cs

```diff
@@ -0,0 +1,48 @@
+using System.Text;
+using System.Text.Json;
+using Microsoft.AspNetCore.Http;
+
+namespace PlatformPlatform.SharedKernel.ApiCore.Middleware;
+
+public class WebAppHandlerMiddleware : IMiddleware
```

Comment:
Seems like this middleware is not registered.

I believe we would need data specific to a self-contained system and user (e.g. feature flags). This would in turn require a database lookup. Having this in a middleware can be less flexible.

Let's explore options for how to inject configuration into the html.

Reply:
Should be fixed now